keystone/server/flask/core.py

```diff
@@ -79,9 +79,20 @@
                       'config_file': '/etc/keystone/watcher.yaml',
                       'include_initiator_user_id_in_metric': 'true',
                       'include_target_project_id_in_metric': 'false'}),
+    # CCloud: add rate_limit middleware
+    _Middleware(namespace='rate_limit.middleware',
+                ep='rate_limit',
+                conf={'config_file': '/etc/keystone/ratelimit.yaml',
+                      'service_type': 'identity',
+                      'rate_limit_by': 'initiator_project_id',
```

Comment:
this is the default - but please also test the results with `initiator_host_address`